pkg/controller/feed/reconcile.go

```diff
@@ -301,6 +301,14 @@ func (r *reconciler) updateFeed(u *feedsv1alpha1.Feed) error {
 	if updated == false {
 		return nil
 	}
+	et := &feedsv1alpha1.EventType{}
+	if err := r.client.Get(context.TODO(), client.ObjectKey{Namespace: feed.Namespace, Name: feed.Spec.Trigger.EventType}, et); err != nil {
+		if errors.IsNotFound(err) {
+			glog.Errorf("Feed %q has unknown event type %q, will not set finalizer", feed.Name, feed.Spec.Trigger.EventType)
```

Comment:
I think this will also prevent setting the `status` now.

Reply:
Which is why I had tried to revert reconcile.go; I'll figure out why that hasn't been done.